conf/machine/raspberrypi5.conf

```diff
@@ -30,3 +34,5 @@ KERNEL_IMAGETYPE_DIRECT ?= "Image"
 KERNEL_BOOTCMD ?= "booti"
 
 UBOOT_MACHINE = "rpi_arm64_config"
+
+RPI_KERNEL_DEVICETREE_OVERLAYS:append = " overlays/bcm2712d0.dtbo"
```

Comment:
Any specific reason why we should have this here as opposed to having it in the base inc file?

Reply:
My only reasoning was that this is Pi5 specific, and installation on other targets would be useless, if it's preferred to instead put it in there (the overhead is minimal of course) then I'm happy to change it

Reply:
I guess for consistency's sake it might be better to also add the required required dtb's to arv8 config & move the overlay append into rbi-base.
Doing that :)